src/components/Table/GridTable.stories.tsx

```diff
@@ -666,6 +667,7 @@ export function ColSpan() {
   return (
     <GridTable<ColspanRow>
       columns={[idCol, nameCol, detailCol, dateCol, priceCol]}
+      style={{ ...condensedStyle, emptyCell: <>&mdash;</> }}
```

Comment:
Wdyt about just putting this directly into our `defaultStyle`? 

Granted, Blueprint could do a `setDefaultStyle` during boot to customize it; I think procurement-frontend did that to get a slightly-different-default-`style`, but I _think_ Blueprint is just using the stock `defaultStyle`...

Reply:
Outside of Blueprint, it seems a bit heavy handed to assume the default style is to always have some sort of content in an empty cell. I wouldn't think that is super apparent to the user.

Reply:
Ah yeah, I guess that's fair. Are you thinking of adding like a `setDefaultStyle` call on the blueprint side, or specifically opt-ing in on a table by table basis?

Reply:
I was originally thinking of opting in table by table, but maybe we want a setDefaultStyle on Blueprint. In my opinion I think Table best practices depend on the content of the table. I think there may be instances where showing the dashes would be incorrect, but maybe not?

Reply:
> In my opinion I think Table best practices depend on the content of the table

Yeah, I don't necessarily disagree, but at least what we're hearing from UX is that "this is for all tables", so I think it'd be interesting to start there, and then see how many exceptions are needed.

And then based on what those exceptions are / how many there are, we can work with UX to define names for the different patterns instead of just "default" or what not.